.github/workflows/codeql.yml

```
@@ -1,98 +1,128 @@
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL Advanced"
+name: "CodeQL Advanced Scan"
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)